.github/workflows/pr-ci.yml

```diff
@@ -73,9 +73,42 @@ jobs:
     with:
       checkout-repo: ${{ needs.checkout-branch.outputs.PR_repo }}
       checkout-ref: ${{ needs.checkout-branch.outputs.PR_head_ref }}
+
+  start-comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Capture Date and Time
+        id: date-time
+        run: echo "::set-output name=date-time::$(date +'%Y-%m-%d %H:%M:%S')"
+      - name: Start Comment
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+            Workflow started at ${{steps.date-time.outputs.date-time}}.
```

Comment:
Is it feasible to add a link to the workflow run here as well? I'm not sure how the link can be retrieved but perhaps there is some Actions API for that or a reusable Action that can help.